### ISSUE TITLE
Limit and skip does not work in a nested query

### ISSUE THREAD

**Issue by jiujiu1123** [orientechnologies/orientdb#2791]
Firstly I am using the version 1.7.8 downloaded from the http://orientechnologies.com. 

What is happening is I try to get all the post that a user's friend have posted
```sql
Select expand(out('posted')) from (select expand(out('friend')) from #12:338)  order by post_time DESC
```
![screen shot 2014-09-08 at 9 37 01 pm](https://cloud.githubusercontent.com/assets/8244605/4219284/9bc6557c-38f8-11e4-9ff5-3623bc526bb8.png)
When I try to add skip and limit in, the order is wrong(not DESC), another thing is limit does not work correctly, as you can see it's only displaying one record but I have set it to be two.
![screen shot 2014-09-08 at 9 37 18 pm](https://cloud.githubusercontent.com/assets/8244605/4219359/36db3f8c-38f9-11e4-902b-38075fc464fd.png)

**Comment by Ndrou** [comment]
A bugfix is planned for the 2.0 ? This is a very embarrasing problem...
Is it linked to #3141 ?
At present, we'll can't complete our project if this bug is not solved...

**Comment by luigidellaquila** [comment]
It's already fixed in 2.1.
Closing, please reopen if you see any related issues

Thanks

Luigi
